README.md

```diff
@@ -7,6 +7,7 @@ Python lib for common ontology operations over a variety of backends.
 [![badge](https://img.shields.io/badge/launch-binder-579ACA.svg)](https://mybinder.org/v2/gh/incatools/ontology-access-kit/main?filepath=notebooks)
 [![Downloads](https://pepy.tech/badge/oaklib/week)](https://pepy.tech/project/oaklib)
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.6456239.svg)](https://doi.org/10.5281/zenodo.6456239)
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](.github/CODE_OF_CONDUCT.md) 
```

Comment:
Later, this document is referred to as the "Code of Conduct". Would it make sense to use that terminology here as well? you could put both:
Contributor Covenant (Code of Conduct)

Reply:
contributor covenant is the name of the document, similar to how "MIT License" is the title on many repository's "LICENSE" file